src/locales/cs.ts

```diff
@@ -0,0 +1,798 @@
+  //   fructuosus_of_braga_martin_of_dume_and_gerald_of_braga_bishops: 'Saints Fructuosus of Braga, Martin of Dume and Gerald of Braga, Bishops',
+  //   fursa_of_peronne_abbot: 'Saint Fursa, Abbot and Missionary',
+  //   gabriel_taurin_dufresse_bishop: 'Saint Gabriel-Taurin Dufresse, Bishop and Martyr',
+  //   gall_of_switzerland_abbot: 'Saint Gall, Abbot and Missionary',
+  //   genevieve_of_paris_virgin: 'Saint Genevieve, Virgin',
+    george_of_lydda_martyr: 'Sv. Jiřího, mučedníka',
+  //   george_preca_priest: 'Saint George Preca, Priest',
+  //   gerard_of_csanad_bishop: 'Saint Gerard of Csanád, Bishop and Martyr',
+  //   germanus_of_auxerre_bishop: 'Saint Germanus of Auxerre, Bishop',
+  //   gertrude_of_nivelles_abbess: 'Saint Gertrude of Nivelles, Abbess',
+    gertrude_the_great_virgin: 'sv. Gertrudy, panny',
+  //   gobnait_of_ballyvourney_virgin: 'Saint Gobnait, Virgin',
+  //   gonsalo_garcia_martyr: 'Saint Gonsalo Garcia, Martyr',
+    gorazd_of_moravia_and_companions: 'Sv. Gorazda a druhů',
+  //   gotthard_of_hildesheim_bishop: 'Saint Gotthard, Bishop',
+    gregory_i_the_great_pope: 'Sv. Řehoře Velikého, papeže a učitele církve',
+    gregory_vii_pope: 'Sv. Řehoře VII., papeže',
+  //   benedict_of_jesus_valdivielso_saez_religious: 'Saint Benedict of Jesus Valdivielso Sáez, Religious and Martyr',
+  //   hedwig_of_poland: 'Saint Hedwig of Poland',
+    hedwig_of_silesia: 'Sv. Hedviky, řeholnice',
+  //   helena_of_constantinople: 'Saint Helena',
+  //   hemma_of_gurk: 'Saint Hemma of Gurk',
+    henry_ii_emperor: 'Sv. Jindřicha',
+  //   henry_of_finland_bishop: 'Saint Henry, Bishop and Martyr',
+  //   hermann_joseph_of_steinfeld_priest: 'Saint Hermann Joseph, Priest',
+  //   hermenegild_the_visigoths_martyr: 'Saint Hermenegild, Martyr',
+    hilary_of_poitiers_bishop: 'sv. Hilaria, biskupa a učitele církve',
+  //   hilda_of_whitby_abbess: 'Saint Hilda, Abbess',
+  //   hildegard_of_bingen_abbess: 'Saint Hildegard of Bingen, Abbess and Doctor of the Church',
+  //   hubert_of_liege_bishop: 'Saint Hubert, Bishop',
+  //   hugh_of_lincoln_bishop: 'Saint Hugh of Lincoln, Bishop',
+    hyacinth_of_poland_priest: 'Sv. Hyacinta, kněze',
+    ignatius_of_antioch_bishop: 'Sv. Ignáce Antiochijského, biskupa a mučedníka',
+    ignatius_of_loyola_priest: 'Sv. Ignáce z Loyoly, kněze',
+  //   ildephonsus_of_toledo_bishop: 'Saint Ildephonsus of Toledo, Bishop',
+  //   illtud_the_knight_abbot: 'Saint Illtud, Abbot',
+    irenaeus_of_lyon_bishop: 'Sv. Ireneje, biskupa a mučedníka',
+  //   irene_of_macedonia: 'Saint Irene',
+    isidore_of_seville_bishop: 'Sv. Izidora, biskupa a učitele církve',
+  //   isidore_the_farmer: 'Saint Isidore',
+  //   ita_of_killeedy_virgin: 'Saint Ita, Virgin',
+  //   ivo_of_kermartin_priest: 'Saint Ivo, Priest',
+    james_apostle: 'Sv. Jakuba, apoštola',
+    james_apostle_patron_of_spain: 'Sv. Jakuba, apoštola a patrona Španělska',
```

Comment:
Usually there's no "a" before patronal titles:
`Sv. Jakuba, apoštola, patrona Španělska`
(I don't repeat this for every patronal title, but it's a general issue)

Reply:
I consider this a non-issue _per se_, though it is a _style issue_. A style that depends on many factors and I have seen with and without a conjunction. Maybe once, romcal will support both (or multiple) styles after personal metadata separation from the celebration names (see #154).